docs/hw/jetson_hookup.md

```diff
@@ -0,0 +1,29 @@
+# iRobot® Create® 3 Hookup Guide for NVIDIA® Jetson™
+
+The iRobot® Create® 3 has a USB-C®[^1] connector implementing a USB 2.0 host capable of 5 V @ 3 A continuous, which can be used to power and communicate with various downstream devices.
```

Comment:
I wonder if this sentence should be on the main hookup page instead of being triplicated on the separated hookup guides? Especially if we leave the main hookup guide page visible.

Reply:
If we move it, it should go probably after line 4 and before the line I proposed.